pom.xml

```diff
@@ -50,6 +50,9 @@
   </dependencies>
   <packaging>pom</packaging>
 
+  <prerequisites>
```

Comment:
Even with maven-enforcer-plugin, Maven is complaining that prerequisites is missing

Reply:
Yes, is a [known problem](https://github.com/mojohaus/versions-maven-plugin/issues/48). Pablo also suffered this issue. Ask him help